docs/toc.yml

```
@@ -194,6 +194,12 @@
       href: core/whats-new/dotnet-core-2-1.md
     - name: What's new in .NET Core 2.0
       href: core/whats-new/dotnet-core-2-0.md
+  - name: Application Compatibility
+    items:
+    - name: Evalute breaking changes in .NET Core
```

I don't think you need to clarify it's .NET Core here in the TOC

```suggestion
    - name: Evaluate breaking changes
```